projects/packages/post-list/src/class-post-list.php

```diff
@@ -104,14 +113,44 @@ public function add_filters_and_actions( $current_screen ) {
 			 * @param boolean Whether we should show the share action for this post type.
 			 * @param string  The current post type.
 			 */
-			apply_filters( 'jetpack_post_list_display_share_action', false, $current_screen->post_type )
+			apply_filters( 'jetpack_post_list_display_share_action', false, $post_type )
```

Comment:
I've noticed that after a quick edit, the Share action disappears from the quick actions row.

It seems this is caused by this filter returning false — which should not happen on WPCOM (see the `wpcom-jetpack-loader`).
I've poked around a little bit, but I couldn't find a good solution. 🤔 

Either case, I think this could be filed as a minor bug, and IMHO shouldn't block the merge of this PR which address a much more serious issue.
@dsas feel free to take a look yourself, in the off-chance my PHP skills are so rusty I'm overlooking something simple.

Reply:
`wpcom_configure_post_list` in wp-content/mu-plugins/0-wpcom-jetpack-loader.php hooks the filter to return true. That happens *after* Post_List::register is ran. On a page request Post_List::register will set-up the quick actions later, on an ajax request it sets them up immediately - which in this case will fail as the filter hasn't been registered yet.

Ideally the fix might  be to identify a better hook to use for ajax requests in Post_List::register so it acts more similarly to a page load. The quick fix however is to register the filter in `wpcom_configure_post_list`  before the config object makes `Post_List::register` run.

Reply:
Couldn't see a better action being ran, so I rearranged the code in `0-wpcom-jetpack-loader.php` in D70220-code

Reply:
It's also happening on Atomic sites, so I guess we need a similar fix on `wpcomsh`?

Reply:
I've opened 787-gh-Automattic/wpcomsh for that